src/anomalib/data/validators/numpy/__init__.py

```diff
@@ -0,0 +1,17 @@
+"""Anomalib Numpy data validators."""
+
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from .depth import NumpyDepthBatchValidator, NumpyDepthValidator
+from .image import NumpyImageBatchValidator, NumpyImageValidator
+from .video import NumpyVideoBatchValidator, NumpyVideoValidator
+
+__all__ = [
+    "NumpyImageBatchValidator",
+    "NumpyImageValidator",
```

Comment:
@ashwinvaidya17, @djdameln, which one would you prefer?
`"NumpyImageValidator"` or `"NumpyImageItemValidator"`